src/components/main/main-dashboard.jsx

```diff
@@ -2,131 +2,98 @@ import React from 'react';
 import { FaHome, FaCalendarAlt, FaArrowDown } from 'react-icons/fa';
 
 const MainDashboard = () => {
-  return (
-    <div className="main-dashboard">
-      <div className="main-dashboard-item">
-        <div className="items">
-          <FaHome size="50px" />
-          <h1 className="dashboard-txt">Dashboard</h1>
-          <div className="dashboard-smalltext">
-            <h6>Home Dashboard</h6>
-          </div>
+    const dashArr = [
+        {
+            textCircle: '16k',
+            strokeDasharray: '130',
+            strokeDashoffset: '1',
+            title: 'Total balance',
+            percent: '+43%',
+        },
+        {
+            textCircle: '194',
+            strokeDasharray: '65%',
+            strokeDashoffset: '50%',
+            title: 'New members',
+            percent: '+65%',
+        },
+        {
+            textCircle: '76',
+            strokeDasharray: '25%',
+            strokeDashoffset: '50%',
+            title: 'New orders',
+            percent: '+25%',
+        },
+    ];
+    return (
+        <div className="main-dashboard">
+            <div className="main-dashboard-item">
+                <div className="items">
```

Comment:
can we make .map of items?

Reply:
I think it's bad idea, becase I must take in array full tags. What do you mean?

Reply:
no, you need to make config object for these blocks and then make a map of them.
with all additional fields, classNames, etc

Reply:
I made this, I think this can be like this